frontend/configuration.properties.js

```diff
@@ -260,6 +260,21 @@
 	exportButtonLabel: 'Export',
   },
 
+  // These properties configure the spinner that is shown when a search is being performed.
+  BusyIndicator: {
```

Comment:
I'm not 100% convinced that the best thing to do in general is to set these in the configuration instead of just setting them on the two instances of the component... but I guess I don't care too strongly about it...1